recipe/meta.yaml

```diff
@@ -17,6 +17,7 @@ build:
 requirements:
   host:
     - pip
+    - numpy
```

Comment:
Now it is required to install numpy while running setup.py (https://github.com/CyberAgent/cmaes/pull/44).